contracts/core/l2/AaveSubProxy.sol

```diff
@@ -105,10 +110,10 @@ contract AaveSubProxy is StrategyModel, AdminAuth, ProxyPermission, CoreHelper {
     ) public {
         (uint32 subId1, uint32 subId2) = parseSubIds(encodedInput[0:8]);
 
-        SubStorageL2(SUB_STORAGE_ADDR).deactivateSub(subId1);
+        SubStorage(SUB_STORAGE_ADDR).deactivateSub(subId1);
 
         if (subId2 != 0) {
-            SubStorageL2(SUB_STORAGE_ADDR).deactivateSub(subId2);
+            SubStorage(SUB_STORAGE_ADDR).deactivateSub(subId2);
         }
```

Comment:
Maybe create two different contracts so we have both L2 SubProxy and L1 SubProxy in repo

Reply:
The interface for deactivateSub is the same on both, so I would keep it this way to minimize bloat